### ISSUE TITLE
Option to display silent video not as GIF

### ISSUE THREAD

**Issue by jeroenpraat** [tootsuite/mastodon#7315]
I made a screencast without sound today. After uploading the video was displayed as a GIF (so without controls and repeating). This was not the desired behavior, but I understand that the only way to avoid this is to add sound.

To avoid this problem I think it's good idea that after uploading a silent video the user gets a checkbox with the option to display it as a GIF (default) or as a regular video (as in this scenario).

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [x] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by jeroenpraat** [comment]
Actually this is related to  #6403, but more like the opposite.

**Comment by mjankowski** [comment]
Still relevant to 4.3?

**Comment by trwnh** [comment]
yes, this is about videos without sound being converted to looping animations

**Comment by marcschulder** [comment]
This matter is also related to accessibility. When users of sign languages post videos of their signing, they usually do so without sound. Not being able to navigate minutes worth of signed conversation is very frustrating. I just convinced my sign language research department to join Mastodon (yay) and this is point of frustration for us.
